test/test.js

```diff
@@ -198,6 +198,7 @@ describe('miller-columns', function() {
       firstItemL1.click()
 
       const firstItemRemove = document.querySelector('#selected-items button')
+      assert.include(firstItemRemove.textContent, "Parenting, childcare and children's services")
```

Comment:
Is it worth asserting that the span has a `govuk-visually-hidden` class applied? Otherwise someone could remove that class and all the tests would still pass.

Reply:
Good call, Chris, I've updated the assertion to include the `govuk-visually-hidden` class.